src/aks-preview/HISTORY.rst

```diff
@@ -12,6 +12,10 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 
+18.0.0b42
++++++++
+* Add comprehensive LocalDNS live tests for valid and invalid configuration scenarios to improve test coverage and error handling validation.
```

Comment:
did you only added live tests? I thought you fixed a bug.

Reply:
We should add code change histories here. No need to add testing changes, since it is not part of the release